docs/source/release.rst

```diff
@@ -7,33 +7,109 @@ Release Notes
+* :support:`1972` Fix BigQuery backend fixture so batting and awards_players fixture re…
+* :support:`1971` Disable BigQuery explicitly in all/test_join.py
+* :feature:`1967` Implement join for PySpark backend
+* :feature:`1952` Validate AsOfJoin tolerance and attempt interval unit conversion
+* :support:`1963` Re-formatting all files using pre-commit hook
+* :support:`1961` Disable codecov report upload during CI builds
+* :support:`1960` Developer doc enhancements
+* :feature:`1943` filter for PySpark backend
+* :feature:`1945` window operations for pyspark backend
+* :support:`1958` Missing geospatial ops for OmniSciDB
+* :feature:`1951` Implement IntervalSub for pandas backend
+* :support:`1950` Remove pandas deprecation warnings
+* :support:`1948` Add developer docs to get docker setup
+* :support:`1949` More informative IntegrityError on duplicate columns
+* :feature:`1942` PySpark backend string and column ops
+* :support:`1928` Improve geospatial literals and smoke tests
+* :support:`1925` PostGIS enhancements
+* :bug:`1933 major` Fixes explain operation
+* :feature:`1913` PySpark backend
+* :bug:`1937 major` Fix incorrect assumptions about attached SQLite databases
+* :bug:`1938 major` Upgrade to JDK11
+* :support:`1866` Rename mapd to omniscidb backend
+* :support:`1926` Fix failing BigQuery tests
+* :feature:`1908` DDL support for Spark backend
+* :support:`1917` Added missing null literal op
+* :feature:`1923` Support timezone aware arrow timestamps
+* :bug:`1903 major` `sql` method doesn't work when the query uses LIMIT clause
+* :feature:`1860` Add shapely geometries as input for literals
+* :bug:`1910 major` Fix union implementation
+* :bug:`1912 major` Fix failing com imports on master
+* :feature:`1858` Add geopandas as output for omniscidb
+* :bug:`1901 major` OmniSci/MapD - Fix reduction for bool
+* :feature:`1885` Spark UDFs
+* :feature:`1871` Add support for Postgres UDFs
+* :bug:`1899 major` Pass scope to grouping execution in the pandas backend
+* :support:`1895` Update link to Presto website
+* :support:`1896` Removing linting from windows
+* :bug:`1888 major` Fix various Spark backend issues
+* :bug:`1891 major` Make Nodes enforce the proper signature
+* :bug:`1893 major` Fix according to bug in pd.to_datetime when passing the unit flag
+* :feature:`1830` Spark tests
+* :support:`1884` Fix link to NUMFOCUS CoC
+* :bug:`1883 major` Fix small formatting buglet in PR merge tool
+* :support:`1882` Added CoC section
+* :bug:`1876 major` Fix the case where we do not have an index when using preceding with intervals
+* :feature:`1807` Spark client
+* :bug:`1872 major` Fixed issues with geo data
+* :feature:`1868` Use pandas rolling apply to implement rows_with_max_lookback
+* :bug:`1869 major` Remove -x from pytest call in linux CI
+* :bug:`1867 major` Fix return type of Struct.from_tuples
+* :support:`1859` Remove pandas exception for rows_with_max_lookback
+* :support:`1856` Move CI pipelines to Azure
```

Comment:
instead of jamming all of the notes together
can i make subsections. for each release 
eg 1.2 1.3 etc

Reply:
do you mean one file for each release note like pandas? if yes, do you have an specific file structure in mind?

Reply:
no you just add something like

```
Version 1.3
------------
```

before each section

pls build the docs and you will see

Reply:
oh I got it. thanks, I will work on that.